src/main/java/springsamurais/toyswapbackend/controller/ListingController.java

```diff
@@ -1,27 +1,87 @@
 package springsamurais.toyswapbackend.controller;
 
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import springsamurais.toyswapbackend.exception.*;
+import springsamurais.toyswapbackend.exception.ListingFailedToSaveException;
+import springsamurais.toyswapbackend.exception.ListingNotFoundException;
+import springsamurais.toyswapbackend.model.*;
+import springsamurais.toyswapbackend.service.listing.ListingServiceImplementation;
+
+import java.util.List;
+
 import springsamurais.toyswapbackend.model.Listing;
 
 
 @RestController
-@RequestMapping
+@RequestMapping("api/v1/listings")
 public class ListingController {
 
-    @GetMapping
-    public void getAllItems() {
+    @Autowired
+    private ListingServiceImplementation listingService;
+
+
+    @GetMapping("")
+    public ResponseEntity<List<Listing>> getAllItems() {
+        List<Listing> listingList = listingService.getAllListings();
+        return new ResponseEntity<>(listingList, HttpStatus.OK);
+    }
+
+    @GetMapping("/{listingID}")
+    public ResponseEntity<?> getListingById(@PathVariable("listingID") Long listingID) {
+        Listing listingFound;
+
+        listingFound = listingService.getListingById(listingID);
```

Comment:
Could this be on 1 line? 
`Listing listingFound = listingService.getListingById(listingID)`

Not major at all, just a thought.